README.md

```diff
@@ -241,6 +241,7 @@
 - 🚀 [VuePress Tailwind CSS Starter](https://github.com/xiaoluoboding/vuepress-tailwind-theme-starter) - A VuePress starter using Tailwind CSS.
 - 🚀 [Gatsby Serif](https://github.com/windedge/gatsby-tailwind-serif) - Gatsby's serif theme using Tailwind CSS.
 - 🚀 [Seminyak Hugo Theme](https://git.habd.as/jhabdas/seminyak) - Hugo theme using Tailwind CSS.
+- 🚀 [Eleventy Starter Boilerplate](https://creativedesignsguru.com/demo/Eleventy-Starter-Boilerplate/eleventy-starter-boilerplate-presentation/) - Eleventy Starter is production-ready with SEO-friendly for quickly starting a blog using Tailwind CSS.
```

Comment:
- It's better if you link directly to the repository. 
- [Is it normal that the site has no layout?](https://i.imgur.com/OnJn8vV.png)
- I changed the description to fit the list better.

```suggestion
- 🚀 [Eleventy Starter](https://github.com/ixartz/Eleventy-Starter-Boilerplate/) - Production-ready, SEO-friendly blog starter using Tailwind CSS.
```